### ISSUE TITLE
A braking change can be specified with ! appended after, not before, the type/scope

### ISSUE THREAD

**Issue by djantea** [zachdaniel/git_ops#20]
As per v1.0 of the [Conventional Commits spec](https://www.conventionalcommits.org/en/v1.0.0), a breaking change commit can be specified either by a `BREAKING CHANGE: ` footer or a ! **after** the required `type` and the optional `scope`. 

The given exemple is:

```
refactor!: drop support for Node 6
```

So we need to accept the above, as opposed to

```
!refactor: drop support for Node 6
```

**Comment by zachdaniel** [comment]
Oof, great catch!

**Comment by zachdaniel** [comment]
#19 was merged and released as version `0.6.4` on hex.

**Comment by zachdaniel** [comment]
Sorry, wrong issue :)

**Comment by zachdaniel** [comment]
Released on hex as version `1.0.0` (breaking change incurred major release bump, and this has been around it might as well be `1.0` :)
